packages/web/src/components/create-playlist/FormActions.tsx

```diff
@@ -51,6 +52,7 @@ export const EditActions = ({
           onClick={onCancel}
           className={styles.cancelButton}
           textClassName={styles.cancelButtonText}
+          buttonType='button'
```

Comment:
should `button` just be default?

Reply:
It's `submit` on most browsers apparently! :o Without this, hitting "enter" deletes the playlist or folder lol.